modules/backup/main.tf

```diff
@@ -113,3 +113,15 @@ resource "aws_backup_selection" "non_production" {
     }
   }
 }
+
+# SNS topic
+resource "aws_sns_topic" "backup_failure_topic" {
+  name = "backup_failure_topic"
+}
+
+# Attaches the SNS topic to the backup vault to subscribe for notifications
+resource "aws_backup_vault_notifications" "example" {
```

Comment:
This is OK, but not needed, since only the CloudWatch formatted messages end up being read by PD and then get posted to slack.
So this one will notify to SNS, but there is nothing there to read it (other than your email notification, which I assume it's temporary only)